src/System.Text.Utf8String/System/Text/Utf8String.cs

```diff
@@ -12,22 +12,22 @@
 namespace System.Text.Utf8
 {
     [DebuggerDisplay("{ToString()}u8")]
-    public class Utf8String
+    public sealed class Utf8String
```

Comment:
What motivated the decision to seal this class?

Reply:
System.String is sealed. Also, if/once we change this type to special runtime type (variable size class), inheriting from this type won't be possible.